samNode/template.yaml

```diff
@@ -145,7 +145,7 @@ Parameters:
   WebHookURL:
     Type: String
     Default: 'default'
-  SQSExpiryQueueURL:
+  SQSExpiryQueue:
```

Comment:
I'm not sure this is correct - there already is a SQSEXPIRY_QUEUE that becomes the URL of the queue itself on the `writePass` function.  Might need to look at the `SQSExpiryQueueURL` param and env var `SQS_EXPIRY_QUEUE_URL` on line 1276 should be completely removed?

Reply:
That makes sense, Instead of changing the name in the template I have updated the workflows to match SQSExpiryQueueURL param in [commit](https://github.com/bcgov/parks-reso-api/pull/416/commits/11b5569303e91fd5ab182a0bf3612dd5e594192c)